src/bundler.zig

```diff
@@ -1435,12 +1435,18 @@ pub const Bundler = struct {
                                     }, prop.key.?.loc),
                                     .value = prop.value.?,
                                 };
+                                var maybe_quoted = name;
```

Comment:
```suggestion
                                var maybe_quoted = MutableString.ensureValidIdentifier(name, allocator) catch return null;
```

Reply:
We already use this function to create the name of temporary variable, and it replaces the hyphens with underscores. With your suggestion the code becomes this:
```JS
var key_with_hyphen = !0;

export {
  key_with_hyphen
};
export default { "key-with-hyphen": key_with_hyphen };
```